components/PersonalisationModal.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Button, Chip, Modal } from "@mui/material";
+import React from "react";
+import { useState, useEffect } from "react";
+
+const modalStyle = {
+  position: "absolute" as "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 800,
+  bgcolor: "background.paper",
+  boxShadow: 24,
+  borderRadius: 2,
+  p: 2,
+};
+
+const modalHeaderStyle = {
+  fontSize: "1.5rem",
+  fontWeight: "bold",
+  margin: 0,
+};
+
+const modalTextStyle = {};
+
+const PersonalisationModal = ({ industries, clearFilters, addFilter }) => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  // handles toggling of the selected industries
+  const [selectedIndustries, setSelectedIndustries] = useState<string[]>([]);
+  const handleIndustryChipToggle = (industry: string) => {
+    if (selectedIndustries.includes(industry)) {
+      setSelectedIndustries(selectedIndustries.filter((i) => i !== industry));
+    } else {
+      setSelectedIndustries([...selectedIndustries, industry]);
+    }
+  };
+
+  const IsChipSelected = (industry: string) => {
+    return selectedIndustries.includes(industry);
+  };
+
+  const handleContinueButtonClick = () => {
+    clearFilters();
+    selectedIndustries.forEach((industry) => {
+      addFilter("industries", industry, "any");
+    });
+    setOpen(false);
+    localStorage.setItem("ModalClosed", "true");
+  };
+
+  // only open the modal if the user hasn't closed it before
+  useEffect(() => {
+    if (localStorage.getItem("ModalClosed") === null) {
+      setOpen(true);
+    }
+  });
```

Comment:
This part means that repeat visitors who would no longer be presented with the questionnaire and can't select an initial few industries unlike on their first visit. This can be confusing for some people, who might expect that those preferences be stored in some form.

For now, could we remove this part of the code and disable this behaviour? Think in future PRs we can look into what kind of behaviour would be more intuitive for users.